test/src/go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/gruntwork-io/go-commons v0.8.0 // indirect
 	github.com/hashicorp/errwrap v1.0.0 // indirect
 	github.com/hashicorp/go-cleanhttp v0.5.2 // indirect
-	github.com/hashicorp/go-getter v1.5.9 // indirect
+	github.com/hashicorp/go-getter v1.5.11 // indirect
 	github.com/hashicorp/go-multierror v1.1.0 // indirect
 	github.com/hashicorp/go-safetemp v1.0.0 // indirect
 	github.com/hashicorp/go-version v1.3.0 // indirect
```

Comment:
it looks like `go.sum` is out of sync with `go.mod`. To fix you can run:

```bash
$ go mod tidy -go=1.16 && go mod tidy -go=1.17
```

Reply:
What makes you say they are out of sync? They look in sync to me? 
https://github.com/cloudposse/terraform-aws-dynamic-subnets/blob/e5843965b370e3eb69d1762b589e795f3a5cf432/test/src/go.mod#L35

https://github.com/cloudposse/terraform-aws-dynamic-subnets/blob/e5843965b370e3eb69d1762b589e795f3a5cf432/test/src/go.sum#L222